src/Config/Config.php

```diff
@@ -5,22 +5,23 @@
 namespace PharIo\ComposerDistributor\Config;
 
 use PharIo\ComposerDistributor\FileList;
+use RuntimeException;
 
 class Config
 {
     /** @var string */
     private $package;
 
-    /** @var \SplFileInfo|null */
+    /** @var string|null */
     private $keyDirectory;
 
     /** @var \PharIo\ComposerDistributor\FileList */
     private $phars;
 
-    public function __construct(string $package, FileList $phars, ?\SplFileInfo $keyDir = null)
+    public function __construct(string $package, FileList $phars, ?string $keyDir = null)
     {
         if (strpos($package, '/') === false) {
-            throw new \RuntimeException('Invalid package name');
+            throw new RuntimeException('Invalid package name');
```

Comment:
Technically not related, but: I can haz Custom Exception?

Reply:
Will fix in next pull request ;)